network/tests/fuzzing.rs

```diff
@@ -137,3 +141,425 @@ async fn fuzzing_post_handshake() {
         node1.write_bytes(&random_bytes).await;
     }
 }
+
+#[tokio::test(flavor = "multi_thread")]
```

Comment:
was `multi_thread` chosen so that these tests are faster or that there is more strain on the test node, so that these fuzz tests serve as stress tests too? as for the speed, I'm not sure if multi-threaded executors can improve the runtime, as the node is likely to immediately reject most messages; if, on the other hand, the runs you've done locally indicate that it is indeed a performance improvement, I'd make a note about it in a comment